src/CloudGraphs.jl

```diff
@@ -49,35 +49,31 @@ end
 
 # --- Common conversion functions ---
 function exVertex2CloudVertex(vertex::ExVertex)::CloudVertex
-  cgvProperties = Dict{String, Any}();
-
   #1. Get the special attributes - payload, etc.
   propNames = keys(vertex.attributes);
-  if("bigData" in propNames) #We have big data to save.
-    bigData = vertex.attributes["bigData"];
-  else
-    bigData = BigData();
-  end
-  if haskey(vertex.attributes, "data") #("data" in propNames) #We have protobuf stuff to save in the node.
-    packed = vertex.attributes["data"];
-  else
-    packed = "";
-  end
+  #We have big data to save.
+  bigData = "bigData" in propNames ? vertex.attributes["bigData"] : BigData();
+  #We have protobuf stuff to save in the node.
+  packed = haskey(vertex.attributes, "data") ? vertex.attributes["data"] : packed = "";
+  labels = haskey(vertex.attributes, "tags") ? vertex.attributes["tags"] : String[];
```

Comment:
double checking that the names cross over here (at least for the now).  In CG and Neo there are things called labels === tags in Graff/IIF/RoME/Caesar